packages/zwave-js/src/lib/controller/Controller.ts

```diff
@@ -1022,11 +1030,13 @@ export class ZWaveController
 						)
 					}`,
 				);
+				this._supportsLongRange = resp == RFRegion["USA (Long Range)"];
 			} else {
 				this.driver.controllerLog.print(
 					`Querying the RF region failed!`,
 					"warn",
 				);
+				this._supportsLongRange = false;
```

Comment:
*sigh* ... I can't tell what's supposed to be the definitive reference for this. I'd REALLY _want_ to use the long range bit in the `SerialAPIStarted` message, but that's sent after a soft reset, and I think we have many cases/sticks (including ser2net) that require we DON'T do a soft reset. This was the next best option I could find.